### ISSUE TITLE
Example code doesn't work

### ISSUE THREAD

**Issue by BethTian** [jadchaar/sec-edgar-downloader#117]
I cannot run the example code in the quick start and I don't know why.
<img width="692" alt="image" src="https://github.com/jadchaar/sec-edgar-downloader/assets/109475450/3a40743f-561a-43a9-af41-8b7c5247cc5f">

**Comment by BethTian** [comment]
@jadchaar I changed computers to try. But the problem still there. I would appreciated it if you can help me.

**Comment by BethTian** [comment]
I want to add some information about the error. It seems like there is something wrong in the `search_query_results`.

![d01f9632cff0fe4507e02fd3ce72f5f](https://github.com/jadchaar/sec-edgar-downloader/assets/109475450/14bb826d-2fe1-4bf9-aedd-152341596725)

**Comment by BethTian** [comment]
I want to update some new findings. This error happened when the number of files on the SEC website is less than that we asked in the code. Hence, if anyone meet this kinds of problems, you can check the number of files in the SEC website and change the number requested. Then it will success. The following is an example:

![02b25330a6a5397d92849ea1c5f2b74](https://github.com/jadchaar/sec-edgar-downloader/assets/109475450/8b868fed-84a5-43f4-b0c1-0adec1b3998a)

But how can I know how many files they have in advance?

**Comment by TimoImo** [comment]
Hey there, the SEC slightly changed the format of their interface's response.

All you need to do is locate the file `_utils.py' of the package.

In the import section add:
```
import json
```
In line 209 replace
```
query_size = search_query_results["query"]["size"]
```
with
```
query_size = json.loands(search_query_results["query"])["size"]
```
The solution is probably not very elegant, but it's working.

**Comment by BethTian** [comment]
Okay. Thank you very much.



TimoImo ***@***.***> 於 2023年8月18日週五 下午4:32寫道：

**Comment by JennyHuang23** [comment]
I face the same problem when I try to get equity information from Form 4. 
![1692858526857](https://github.com/jadchaar/sec-edgar-downloader/assets/127916986/b22a31d5-eae5-4ee0-8c87-e0fe65b66233)
Do you know how to solve this problem, please?

**Comment by TimoImo** [comment]
Hi @JennyHuang23 , if you follow the steps I described in my earlier post, you should be able to fix the error.

**Comment by jadchaar** [comment]
Hi all, this should now be fixed in v5.0 that I released today. In v4.x of this package, I was using a reverse-engineered private SEC API, but I have now migrated and fully re-written the package to use the official SEC Edgar API - https://www.sec.gov/edgar/sec-api-documentation. Please do check out the update!
